code/migration/src/main/scala/com/namely/chiefofstate/migration/versions/v1/V1.scala

```diff
@@ -137,45 +125,35 @@ case class V1(
    * can be used sql statement.
    */
   private def transformTableName(): String = {
-    if (offsetStoreTable.isUpper) offsetStoreTable.quote
-    else offsetStoreTable
+    if (priorOffsetStoreTable.isUpper) priorOffsetStoreTable.quote
+    else priorOffsetStoreTable
   }
 }
 
 object V1 {
   // offset store temporary table name
   private[v1] val tempTable: String = "v1_offset_store_temp"
+  // offset store final table name
+  private[v1] val offsetTableName: String = "read_side_offsets"
 
   /**
    * creates the temporary offset store table in the write side database
    *
    * @param journalJdbcConfig the database config
    */
   private[v1] def createTable(journalJdbcConfig: DatabaseConfig[JdbcProfile]): Unit = {
-    val tableCreationStmt: SqlAction[Int, NoStream, Effect] =
-      sqlu"""
-        CREATE TABLE IF NOT EXISTS #$tempTable(
+    val stmt = sqlu"""
+        CREATE TABLE #$tempTable(
           projection_name VARCHAR(255) NOT NULL,
           projection_key VARCHAR(255) NOT NULL,
           current_offset VARCHAR(255) NOT NULL,
           manifest VARCHAR(4) NOT NULL,
           mergeable BOOLEAN NOT NULL,
           last_updated BIGINT NOT NULL,
           PRIMARY KEY(projection_name, projection_key)
-        )"""
-
-    val indexCreationStmt: SqlAction[Int, NoStream, Effect] =
```

Comment:
Ay reason why we don't have the index on the temp table? The temp table is an identical copy of the actual read side offset store table. If it works that is fine.

Reply:
you were deleting this index anyways before recreating it. it is not functional for the migration, so no point in keeping it